### PR TITLE
Adding limiter to oblique OBC

### PR DESCRIPTION
**Pull request by kshedstrom**
Adding the normal phase speed limiter from ORLANSKI to OBLIQUE. It matters!

Also fussing with Kelvin wave setup.

### CODE REVIEW

**Comment by marshallward** [comment]
Thanks Kate, can you update this against the latest `dev/gfdl`?  Normally we can do this on our end, but it's currently not letting me.

Also, have you had a look at #1058 and #1060?  Do you think these may affect your runs?  (#1058 has been merged but #1060 has not)

**Comment by kshedstrom** [comment]
Thanks, Marshall. I updated this branch against dev/gfdl. Some recent changes have been changing answers ever so subtilely, including your #1060 which I had not previously noticed. I haven't redone any of those skill test sweeps with the latest code yet.

File: src/core/MOM_open_boundary.F90 [hunk 1]
```diff
@@ -1962,6 +1962,7 @@ subroutine radiation_open_bdry_conds(OBC, u_new, u_old, v_new, v_old, G, US, dt)
            if (dhdt*dhdx < 0.0) dhdt = 0.0
            rx_new = dhdt*dhdx
            cff_new = max(dhdx*dhdx + dhdy*dhdy, eps)
+           rx_new = min(rx_new, cff_new*rx_max)
```
Comment (Hallberg-NOAA) [inline src/core/MOM_open_boundary.F90]: These changes look good to me, but a slightly more parsimonious and bitwise equivalent approach would be to merge line 1963 into the first argument of line 1965, and similarly for the other analogous 3 cases later in the code.

**Comment by marshallward** [comment]
I was sitting on this due to the removal of the `answers2018` blocks, but @Hallberg-NOAA pointed out to me that you (@kshedstrom) would have been the only one impacted by those old buggy versions anyway, so I am happy to merge this.  Up to you if you want to make the change suggested by Bob.

**Comment by marshallward** [comment]
Sorry @kshedstrom I need you to do one more merge against dev/gfdl and then I can test and merge this.

I'll talk with @adcroft on Monday to figure out why I cannot do this on my end.

**Comment by kshedstrom** [comment]
I will be happy to include @Hallberg-NOAA 's suggestion. I was also thinking of interleaving oblique and orlanski to reduce a few lines of code.

**Comment by marshallward** [comment]
Gaea regression test: https://gitlab.gfdl.noaa.gov/ogrp/MOM6/pipelines/9902

**Comment by marshallward** [comment]
No regressions; I will go ahead and merge this.  Thanks @kshedstrom !

**Pull request merged**